.azure-pipelines/all.yml

```diff
@@ -135,10 +138,10 @@ jobs:
       displayName: Puma
       os: linux
     - checkName: purefa
-      displayName: PureFA
+      displayName: purefa
```

Comment:
Dot this: https://github.com/DataDog/integrations-extras/pull/1365#discussion_r918605434 and try again...
Now it is failing because:
```
import fiddler as fdl
E ModuleNotFoundError: No module named 'fiddler'
```
After you add your new dependency as indicated in https://github.com/DataDog/integrations-extras/pull/1365#discussion_r918605434 it should not fail...

Reply:
@rajeevthefiddler You shouldn't need to change `PureFA` or `PureFB`. I remember these were originally lower case and kept failing validations for other contributors. See https://github.com/DataDog/integrations-extras/pull/1331. Does the error go away if you keep the `displaynames` as `PureFA` and `PureFB`?